composer.json

```diff
@@ -11,23 +12,39 @@
     }
   ],
   "require": {
-    "php": ">=5.3.2",
-    "contao/core-bundle": "~3.2 || ^4.1",
-    "contao-community-alliance/composer-plugin": "~2.0 || 3.*",
+    "php": "^7.1",
+    "contao/core-bundle":"^4.4",
     "codefog/contao-haste": "4.*"
   },
+  "require-dev": {
+    "contao/manager-plugin": "^2.0"
+  },
+  "conflict": {
+    "contao/core": "*",
+    "contao/manager-plugin": "<2.0 || >=3.0"
+  },
   "replace": {
     "contao-legacy/tablelookupwizard": "self.version"
   },
+  "autoload": {
+    "psr-4": {
+      "Terminal42\\ContaoTableLookupWizard\\": "src/"
+    },
+    "classmap": [
```

Comment:
All classes should be moved to `src` so that you don't need `classmap` and `exclude-from-classmap`.

Reply:
This is what is done, although contao/news-bundle does also that. What's the reason then ?
https://github.com/contao/news-bundle/blob/master/composer.json#L41

Reply:
Yes, it's nonsense. I mean, all the core related bundles like news, calendar etc. have been transformed like what you did here but as I said: it makes no sense to do so. It only makes sense if you want to create a new version and moving files around, re-arranging namespaces, introducing new features etc. Contao couldn't do that because we wanted to make the upgrade path from 3.5 to 4.0 easy which means you cannot just move e.g. `\ModuleNewslist` to e.g. `\Contao\NewsBundle\FrontendModule\List` because every module/extension that did an `extends ModuleNewslist` would not work anymore.

Reply:
I see my mistake then, following a bundle that was made from a Contao 3 extension who also wants to keep the retro-compatibility.
Guess I'll look more closely at real Symfony bundle made for Contao 4+ (seems you have few already in your org).